# request
time timestamp
---
# response
OrientationStamped orientation
